mcs/class/System.ServiceModel/System.ServiceModel.Channels.Http/HttpListenerManager.cs

```diff
@@ -48,24 +48,29 @@ protected HttpListenerManager ()
 			Entries = new List<HttpChannelListenerEntry> ();
 		}
 
-		public List<HttpChannelListenerEntry> Entries { get; private set; }
+		protected List<HttpChannelListenerEntry> Entries { get; private set; }
```

Comment:
Why not `private`? What if other code outside this class tries to access it without locking?